File: ServerSide/src/main/java/com/tiago/coelho/NoteApp/dao/Note.java
package com.tiago.coelho.NoteApp.dao;

public class Note {

    private int id;
    private String description;
    private int user_id;

    public Note() {}

    public Note(String description, int user_id) {
        this.description = description;
        this.user_id = user_id;
    }
}

File: ClientSide/AwesomeNotes/Modules/Main/Models/Note.swift
//
//  Note.swift
//  AwesomeNotes
//
//  Created by Tiago Coelho on 16/08/2023.
//

import Foundation

struct Note: Identifiable, Codable {
    let id: Int
    let title: String
    let description: String
    let createdAt: String
    let username: String
    let latitude: String?
    let longitude: String?
}
